index.html

```diff
@@ -107,6 +107,33 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+                <!-- ________ Raj-Pulapakura card START ________  -->
+                <div class="card">
+                  <p class="name">Raj Pulapakura</p>
+                  <p class="contact">
+                    <i class="fab fa-twitter"></i>
+                    <a href="https://www.twitter.com/" target="_blank">I don't use Twitter lol</a>
```

Comment:
```suggestion
```

Remove these 2 line if you don't use twitter.

Reply:
Thanks for the tip.